src/main/java/bio/terra/cli/command/app/passthrough/Git.java

```diff
@@ -0,0 +1,87 @@
+package bio.terra.cli.command.app.passthrough;
+
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Resource;
+import bio.terra.cli.exception.PassthroughException;
+import bio.terra.cli.exception.UserActionableException;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+@Command(name = "git", description = "Call git in the Terra workspace.")
+public class Git extends ToolCommand {
+
+  @CommandLine.Option(
+      names = "--resource",
+      description =
+          "name of the git-repo resources in the current workspace to be cloned. \n"
+              + "Example usage: git clone --resource=repo1 --resource=repo2")
+  public String[] names;
+
+  @CommandLine.Option(
+      names = "-a",
+      description = "clone all the git repo resources in the current workspace")
+  public boolean cloneAll;
+
+  @Override
+  public String getExecutableName() {
+    return "git";
+  }
+
+  @Override
+  public String getInstallationUrl() {
+    return "https://git-scm.com/book/en/v2/Getting-Started-Installing-Git";
+  }
+
+  @Override
+  protected void executeImpl() {
+    workspaceOption.overrideIfSpecified();
+    command.add(0, "git");
+    if (cloneAll && names != null && names.length > 0) {
+      throw new UserActionableException(
+          "Conflicted input argument. Only specify one depending on "
+              + "whether you'd like to clone all or some of the git repo resources in this workspace");
+    }
+    if (cloneAll) {
+      validateCloneCommand();
+      Context.requireWorkspace().getResources().stream()
+          .filter(resource -> Resource.Type.GIT_REPO == resource.getResourceType())
+          .forEach(resource -> cloneGitRepoResource(resource));
```

Comment:
You need to check for duplicates in the repo names. Probably want to check the file system for a directory of the same name (e.g. if a user tries to clone the same  repo twice). Might also want an argument to specify the directory name of the clone (in case they have a legitimate desire to clone something more than once).

Reply:
it should be handled by git itself. if the file already exists, git will throw error and we will just go on to the next one.

Reply:
So you won't throw a `SystemException` if one of the invocations of `cloneGitRepoResource()` gives a non-zero exit code?

Reply:
yeah. error output from git will still be printed, but we should proceed to the next git resources if there's any.